__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const app = require("../app");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => connection.end());
+
+describe("GET /api/topics", () => {
+  test("returns 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("returns an array of topic objects with the correct properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .then(({ body }) => {
+        const expected = {
+          description: "The man, the Mitch, the legend",
+          slug: "mitch",
+        };
+        expect(typeof body).toBe("object");
+        expect(body.topics[0]).toEqual(expected);
+        expect(body.topics).toHaveLength(3);
+      });
```

Comment:
Almost there :) 
Please could you do a forEach to check that all of the topics are valid rather than just index 0